### ISSUE TITLE
Incorrect alias when lookUpTable=>alias_field is longer than 256 characters

### ISSUE THREAD

**Issue by MikeVanDeeAvrotros** [dmitryd/typo3-realurl#533]
Url is incorrect when the lookUpTable=>alias_field value is longer than 256(Max mysql alias length) characters. 
See UrlEncoder.php line 418.
`$aliasValue = $this->tsfe->csConvObj->substr('utf-8', $row[$configuration['alias_field']], 0, $maxAliasLengthLength);`
The first index in `$row` is only 256 characters long. So `$aliasValue` is empty string.

https://dev.mysql.com/doc/refman/5.7/en/identifiers.html

**Comment by dmitryd** [comment]
Yes, that's right. 256 characters limit. Make your aliases smaller :)

**Comment by MikeVanDeeAvrotros** [comment]
I can't make my alias shorter.
Is it possible to add `AS alias_value` in the query? So I can make my lookUpTable=>alias_field value  longer than 256.
